rpm/stackdriver-agent.spec

```diff
@@ -11,6 +11,7 @@
 %define _sysconfdir %{_prefix}/etc
 %define _mandir %{_prefix}/man
 %define _initddir /etc/rc.d/init.d
+%define _confdir /etc/stackdriver
```

Comment:
Two small things:
1. Maybe move this right next to _sysconfdir so that the config directory info is close together in this file.
2. Would you please "fix" the comment above ("a little unorthodox but...") to reflect your changes here?

Reply:
Done

Reply:
IMO, once you've done 2, you don't really need to do 1.